Update Upgrade question and syntax on FAQ.md (#48)
ghostbsd/documentation: source/user/FAQ.md

### What is GhostBSD?

> Briefly, GhostBSD is a UNIX®-like operating system. It works on AMD64 and Intel® platforms. GhostBSD is based on FreeBSD, which is based on U.C. Berkeley’s “4.4BSD-Lite” release, with some “4.4BSD-Lite2” enhancements. It is also based indirectly on William Jolitz’s port of U.C. Berkeley’s “Net/2” to the i386, known as “386BSD”, though very little of the 386BSD code remains. Currently, GhostBSD provides MATE desktop as the default version and a community release with XFCE desktop.

Reviewer: Lynne Jolitz also helped quite a bit with 386BSD so we should give her credit too, I think.

For most users, yes.

Most people do not use an operating system. Instead, they use applications that make a computer useful. GhostBSD is designed to provide a desktop and full-featured environment for applications. It supports various web browsers, office suites, email readers, graphics programs, programming environments, network servers, and everything else you might want. Most of these applications can be built using the FreeBSD Ports Collection.

> Suppose you need to use an application only available on an operating system will be hard to replace that operating system (this situation is called vendor lock-in). However, the chances are that there is a very similar application on GhostBSD.

Reviewer: The first sentence sounds weird to me.

```suggestion
If you need to use an application only available on a specific operating system, it may be hard to replace that operating system (this situation is called vendor lock-in). However, chances are that there is a very similar alternative application on GhostBSD.
```
